__tests__/app.test.js

```diff
@@ -38,4 +39,35 @@ describe("GET/topics", () => {
             expect(topicsArray[2].description).toBe('what books are made of')
         })
     })
+})
+
+describe("GET/api", () => {
+    test("200: when calling the api as the endpoint, return an object containing all of the endpoint information for every endpoint tested in app.js", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            const { endpointObject } = body
+            return fs.readFile('./endpoints.json', 'utf-8')
```

Comment:
You could require the endpoints file into the test, as it's a js / json file it's already formatted correctly... readFile is also fine however